src/modules/navigation/NavigationViewContainer.js

```diff
@@ -1,23 +1,22 @@
 import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
 import {pushRoute, popRoute, switchTab, navigationCompleted} from './NavigationState';
 import NavigationView from './NavigationView';
 
 export default connect(
   state => ({
     navigationState: state.get('navigationState').toJS()
   }),
-  dispatch => ({
-    switchTab(key) {
-      dispatch(switchTab(key));
-    },
-    pushRoute(index) {
-      dispatch(pushRoute(index));
-    },
-    onNavigateBack() {
-      dispatch(popRoute());
-    },
-    onNavigateCompleted() {
-      dispatch(navigationCompleted());
-    }
-  })
+  dispatch => {
+    return {
+      switchTab: bindActionCreators(switchTab, dispatch),
+      pushRoute: bindActionCreators(pushRoute, dispatch),
+      onNavigateBack: bindActionCreators(popRoute, dispatch),
+      onNavigateCompleted() {
+        // FIXME: why is navigationCompleted non-existant in NavigationState?
```

Comment:
Would like to get this resolved first before merging it. Since we're not using it why can't we just drop it here? Another option would be to leave it for now since we might replace navigation soon with `react-navigation`